docassemble/MassAccess/data/questions/massaccess.yml

```
@@ -67,4 +68,76 @@ code: |
+
+  <h3 class="h5">To use this site</h3>
+
+  * You need to be over 13 to use this site.
+  * This site is for **anyone** 13 and over. For example: you can be the person who needs the help, a lawyer, or a social worker.
+  * You may not sell the information or the contents of this site. But you can charge for your own time.
+  * Please keep your use fair. Do not use this site in a way that is illegal or that makes it harder for other people to use.
+
+  <h3 class="h5">Information we and others collect</h3>
+
+  * We collect the information that you type to help you complete your forms. We delete all information 180 days after you last update it. You can also delete information immediately.
+  * We log information including IP addresses and web browsers from all visitors. We use this information to keep our site secure. We keep logs for up to 180 days.
+  * We use Google Maps to help fill in addresses automatically. This feature sends your IP address to Google Maps to get your approximate location.
+  * We use Google Analytics to learn how people use our website. This helps us understand which pages are hardest to use. Google may use this information to show you better advertisements.
+  * We use email and text message delivery services that may keep their own records of any messages you send. If you choose to log in with your phone number, this may include a record of the times you log in to the site.
+
+  <h3 class="h5">We keep your information safe</h3>
+
+  * We use the same technology that banks and online stores use to keep your information safe. We use:
+  * 256 bit encryption
+  * Software firewalls
+  * Limited access controls
+  * Continual monitoring
+  * Regular security updates
+
+  <h3 class="h5">Sharing your information</h3>
+
+  * We never sell your information.
+  * We share anonymous information with a small number of nonprofits and researchers. We use the information to improve our site and to improve free legal help in Massachusetts.
+  * When you visit some pages of the site, the site may ask if you want to apply to get a free lawyer from legal aid. You get to decide if you want to share your personal information.
+  * You may choose to email completed forms or share a link to your in-progress work at any time to any person.
+
+  <h3 class="h5">These terms of use can change</h3>
+
+  We may change these Terms of Use at any time with no notice. Please review the terms on a regular basis to see any changes.
+
+  If you use the site after we make changes, you accept our changed Terms of Use.
+
+  <h3 class="h5">The laws of Massachusetts apply to these Terms of Use</h3>
+
+  If a judge decides that one of the terms in this Terms of Use is unenforceable, it will not change the terms that remain.
+
+  <h3 class="h5">If you still have questions</h3>
+
+  Send any questions to:
+
+  Suffolk Legal Innovation and Technology Lab
+  120 Tremont Street Suite 685
+  Boston, MA 02108
```

```suggestion
  Suffolk Legal Innovation and Technology Lab  
  120 Tremont Street Suite 685  
  Boston, MA 02108
```
Not a huge fan of Markdown's extra spaces at the end of lines, but they are needed to put things on new lines in this case. Can also use `<br>`.